web/satellite/src/components/account/billing/paymentMethods/TokenTransactionItem.vue

```diff
@@ -0,0 +1,169 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+<template>
+    <div class="container">
+        <div class="divider" />
+        <div class="container__row">
+            <div class="container__row__item__date-container">
+                <p class="container__row__item date">{{ billingItem.start.toLocaleDateString() }}</p>
+                <p class="container__row__item time">{{ billingItem.start.toLocaleTimeString([], {hour: '2-digit', minute: '2-digit'}) }}</p>
+            </div>
+            <div class="container__row__item__description"> 
+                <p class="container__row__item__description__text">CoinPayments {{ billingItem.description.includes("Deposit")? "Deposit": "Withdrawal" }}</p>
+                <p class="container__row__item__description__id">{{ billingItem.id }}</p>
+            </div>
+            <p class="container__row__item amount">
+                <b>
+                    <span v-if="billingItem.type === 1">
```

Comment:
if it is an enum lets use concrete value of it, not magic number

Reply:
This is following the original design pattern of the component this was based on.